specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json

```diff
@@ -3103,6 +3223,237 @@
+            },
+            "status": {
+              "type": "string",
+              "description": "Enumerates the possible values for the status of the Event Hub.",
+              "enum": [
+                "Active",
+                "Disabled",
+                "Restoring",
+                "SendDisabled",
+                "ReceiveDisabled",
+                "Creating",
+                "Deleting",
+                "Renaming",
+                "Unknown"
+              ],
+              "x-ms-enum": {
+                "name": "EntityStatus",
+                "modelAsString": false
+              }
+            },
+            "captureDescription": {
+              "$ref": "#/definitions/CaptureDescription",
+              "description": "Properties of capture description"
+
+            }
+          },
+          "description": "Properties supplied to the Create Or Update Event Hub operation."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Single item in List or Get Event Hub operation"
+    },
+    "EventHubListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Eventhub"
+          },
+          "description": "Result of the List EventHubs operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of EventHubs."
+        }
+      },
+      "description": "The result of the List EventHubs operation."
+    },
+    "CaptureDescription": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "A value that indicates whether capture description is enabled. "
+        },
+        "encoding": {
+          "type": "string",
+          "description": "Enumerates the possible values for the encoding format of capture description.",
+          "enum": [
+            "Avro",
+            "AvroDeflate"
+          ],
+          "x-ms-enum": {
+            "name": "EncodingCaptureDescription",
+            "modelAsString": false
+          }
+        },
+        "intervalInSeconds": {
+          "format": "int32",
+          "type": "integer",
+          "maximum": 900,
+          "minimum": 60,
+          "description": "The time window allows you to set the frequency with which the capture to Azure Blobs will happen, value should between 60 to 900 seconds"
+        },
+        "sizeLimitInBytes": {
+          "format": "int32",
+          "type": "integer",
+          "maximum": 524288000,
+          "minimum": 10485760,
+          "description": "The size window defines the amount of data built up in your Event Hub before an capture operation, value should be between 10485760 to 524288000 bytes"
```

Comment:
nit: "...a capture operation, value should be between 10485760 and 524288000 bytes"